packages/firebase_core/firebase_core_web/lib/src/interop/core_interop.dart

```diff
@@ -8,11 +8,13 @@
 @JS('firebase_core')
 library firebase_interop.core;
 
+import 'dart:js_interop';
+
 import 'package:firebase_core_web/firebase_core_web_interop.dart';
-import 'package:js/js.dart';
 
 @JS()
-external List<AppJsImpl> getApps();
+// List<AppJsImpl>
+external JSArray getApps();
```

Comment:
Once `AppJsImpl` is moved to an extension type that implements `JSObject`, this can be `JSArray<AppJsImpl>`, and a cast won't be needed above, but for now this is the best alternative. :/